packages/integration-react/package.json

```diff
@@ -11,21 +11,18 @@
     "url": "https://github.com/grafana/grafana-javascript-agent.git",
     "directory": "packages/integration-react"
   },
-  "type": "module",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "build/cjs/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/cjs/index.d.ts",
   "files": [
-    "dist"
+    "build",
+    "README.md",
+    "LICENSE"
   ],
-  "imports": {
```

Comment:
IIRC the `imports` property was there for allowing imports from the build folder instead of the source folder. I had issues in the beginning with this.